docs/web/howto/listings/client/cookies.py

```diff
@@ -1,5 +1,7 @@
+from http.cookiejar import CookieJar
+
 from twisted.internet import reactor
-from twisted.python import compat, log
```

Comment:
can this PR also remove the long deprecated compat alias?

https://github.com/twisted/twisted/blob/311d8dbaf40ddeb887cd3edab1333a3180e9d4d9/src/twisted/python/compat.py#L597-L602

I think that we are aready past 1 year and 2 releases

Reply:
I would like to separate out that work, in case it needs to be reverted for some reason, it's cleaner to have it as a separate single trunk merge commit.